#!/usr/bin/env python
PACKAGE = "saliency_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("min_major_axis", double_t, 0, "Minimum major axis of object to consider", 0.04, 0, 0.3)
gen.add("max_major_axis", double_t, 0, "Maximum major axis of object to consider", 0.12, 0, 0.3)

exit(gen.generate(PACKAGE, "ground_projector", "ground_projector_params"))
